From c3fd0e999a30673712ef6d5db518057bf8996726 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Mon, 8 Jun 2015 14:43:24 +0200
Subject: [PATCH] Initial commit

commit 0a515a50c2aa1ce0f77563a22c719bfd58ecade7

    Disables resourceqt-client build
---
 libresourceqt.pro | 2 --
 1 file changed, 2 deletions(-)

diff --git a/libresourceqt.pro b/libresourceqt.pro
index 60a4789..1ba4807 100644
--- a/libresourceqt.pro
+++ b/libresourceqt.pro
@@ -24,11 +24,9 @@ TEMPLATE = subdirs
 SUBDIRS = libdbus-qeventloop  \
           libresourceqt	      \
           libmediaoverridesqt \
-          resourceqt-client   \
           tests

 tests.depends = libdbus-qeventloop libresourceqt
-resourceqt-client.depends = libdbus-qeventloop libresourceqt


 distribution.commands   = ./makedist.sh
